sound/hybrid_font.h

```diff
@@ -8,11 +9,12 @@ class IgniterConfigFile : public ConfigFile {
     CONFIG_VARIABLE(volHum, 15);
     CONFIG_VARIABLE(volEff, 16);
     CONFIG_VARIABLE(ProffieOSSwingSpeedThreshold, 250.0f);
-    CONFIG_VARIABLE(ProffieOSSwingVolumeSharpness, 1.0f);
-    CONFIG_VARIABLE(ProffieOSMaxSwingVolume, 3.0f);
+    CONFIG_VARIABLE(ProffieOSSwingVolumeSharpness, 0.5f);
```

Comment:
Updated defaults for accent swings to tested values that produce a much better sound for fonts that don't have values set in config.ini

Reply:
This is something that we should try to avoid doing as much as possible.
Changing the defaults means that we may cause problems for some people.
Are you SURE these new values are better?

Reply:
yes, did quite a bit of testing with Dmitry and Jesse Kirkbride.